rebekka-source/Rebekka/StreamOperation.swift

```diff
@@ -70,6 +70,7 @@ internal class StreamOperation: Operation, NSStreamDelegate {
     }
     
     func finishOperation() {
+        NSThread.sleepForTimeInterval(0.1)
```

Comment:
It was my rather "potato" solution for fixing occasionally crashes when queuing many downloads. The problem was that sometimes to stream still receives/transfers data and `self.currentStream?.close()` cuts the stream.
It sometimes happened that you got a `BAD_EXEC`, but sadly no 100% reproducible. 
I am sure that there must be another solution for this, but it needs further testing.